### PR TITLE
Update Kotlin to 1.9.10

### PR DESCRIPTION
**Pull request by qwwdfsad**

### CODE REVIEW

File: core/build.gradle.kts [hunk 1]
```diff
@@ -88,7 +88,7 @@ kotlin {
 
     }
 
-    js {
+    js(IR) {
```
Comment (lppedd) [inline core/build.gradle.kts:91]: @FenstonSingel can I ask why you specified IR explicitly? Isn't the default IR now?
Just curious as in my libraries I'm now omitting it.

Comment (ilya-g) [reply core/build.gradle.kts:91]: Yes, it shouldn't be needed. Instead, the gradle property `kotlin.js.compiler=both` should be removed.

**Comment by qwwdfsad** [comment]
@ilya-g PTAL

File: kotlin-js-store/yarn.lock
Comment (ilya-g) [inline kotlin-js-store/yarn.lock:1]: Should it be regenerated after removal?

**Review by ilya-g — CHANGES_REQUESTED** [review]

**Comment by ilya-g** [comment]
Obsoleted by #318

**Pull request closed without merging**